commit 9e09454cdefc14397c57b2435d3da0c0413dbb8e
Author: Ake Sandgren <ake.sandgren@hpc2n.umu.se>
Date:   Thu Feb 27 15:18:52 2025 +0100

    Avoid IS_SANDCASTLE tripping on //caffe2/test/cpp/jit:test_custom_class_registrations

diff --git a/test/export/test_export.py b/test/export/test_export.py
index 9d1006745b4..b7ee8c1d451 100644
--- a/test/export/test_export.py
+++ b/test/export/test_export.py
@@ -3917,7 +3917,7 @@ class TestExportCustomClass(TorchTestCase):
     def setUp(self):
         if IS_FBCODE:
             lib_file_path = "//caffe2/test/cpp/jit:test_custom_class_registrations"
-        elif IS_SANDCASTLE or IS_MACOS:
+        elif False or IS_MACOS:
             raise unittest.SkipTest("non-portable load_library call used in test")
         elif IS_WINDOWS:
             lib_file_path = find_library_location('torchbind_test.dll')
diff --git a/test/export/test_lift_unlift.py b/test/export/test_lift_unlift.py
index edbfa0803d4..a93728f8bd3 100644
--- a/test/export/test_lift_unlift.py
+++ b/test/export/test_lift_unlift.py
@@ -146,7 +146,7 @@ class TestLift(TestCase):
     def setUp(self):
         if IS_MACOS:
             raise unittest.SkipTest("non-portable load_library call used in test")
-        elif IS_SANDCASTLE or IS_FBCODE:
+        elif False or IS_FBCODE:
             torch.ops.load_library(
                 "//caffe2/test/cpp/jit:test_custom_class_registrations"
             )
@@ -379,7 +379,7 @@ class ConstantAttrMapTest(TestCase):
     def setUp(self):
         if IS_MACOS:
             raise unittest.SkipTest("non-portable load_library call used in test")
-        elif IS_SANDCASTLE or IS_FBCODE:
+        elif False or IS_FBCODE:
             torch.ops.load_library(
                 "//caffe2/test/cpp/jit:test_custom_class_registrations"
             )
diff --git a/test/export/test_serialize.py b/test/export/test_serialize.py
index 08381d95b44..59016a874ad 100644
--- a/test/export/test_serialize.py
+++ b/test/export/test_serialize.py
@@ -823,7 +823,7 @@ class TestSaveLoad(TestCase):
 @unittest.skipIf(not torchdynamo.is_dynamo_supported(), "dynamo doesn't support")
 class TestSerializeCustomClass(TestCase):
     def setUp(self):
-        if IS_SANDCASTLE or IS_FBCODE:
+        if False or IS_FBCODE:
             torch.ops.load_library(
                 "//caffe2/test/cpp/jit:test_custom_class_registrations"
             )
diff --git a/test/export/test_torchbind.py b/test/export/test_torchbind.py
index 362064812d4..e5b2f4ae511 100644
--- a/test/export/test_torchbind.py
+++ b/test/export/test_torchbind.py
@@ -21,7 +21,7 @@ class TestExportTorchbind(TestCase):
     def setUp(self):
         if IS_MACOS:
             raise unittest.SkipTest("non-portable load_library call used in test")
-        elif IS_SANDCASTLE or IS_FBCODE:
+        elif False or IS_FBCODE:
             torch.ops.load_library(
                 "//caffe2/test/cpp/jit:test_custom_class_registrations"
             )
diff --git a/test/export/test_unflatten.py b/test/export/test_unflatten.py
index 64f123136ba..d0baef3777c 100644
--- a/test/export/test_unflatten.py
+++ b/test/export/test_unflatten.py
@@ -552,7 +552,7 @@ class TestUnflatten(TestCase):
     def test_unflatten_constant_obj(self):
         if IS_MACOS:
             raise unittest.SkipTest("non-portable load_library call used in test")
-        elif IS_SANDCASTLE or IS_FBCODE:
+        elif False or IS_FBCODE:
             torch.ops.load_library(
                 "//caffe2/test/cpp/jit:test_custom_class_registrations"
             )
diff --git a/test/higher_order_ops/test_with_effects.py b/test/higher_order_ops/test_with_effects.py
index 0c8f12a1058..3347b6ca1fc 100644
--- a/test/higher_order_ops/test_with_effects.py
+++ b/test/higher_order_ops/test_with_effects.py
@@ -27,7 +27,7 @@ class TestWithEffects(TestCase):
     def setUp(self):
         if IS_MACOS:
             raise unittest.SkipTest("non-portable load_library call used in test")
-        elif IS_SANDCASTLE or IS_FBCODE:
+        elif False or IS_FBCODE:
             torch.ops.load_library(
                 "//caffe2/test/cpp/jit:test_custom_class_registrations"
             )
diff --git a/test/test_weak.py b/test/test_weak.py
index cc9ff6f58ee..54ee73a6dae 100644
--- a/test/test_weak.py
+++ b/test/test_weak.py
@@ -600,7 +600,7 @@ class WeakKeyDictionaryScriptObjectTestCase(TestCase):
 
     def __init__(self, *args, **kw):
         unittest.TestCase.__init__(self, *args, **kw)
-        if IS_SANDCASTLE or IS_FBCODE:
+        if False or IS_FBCODE:
             torch.ops.load_library(
                 "//caffe2/test/cpp/jit:test_custom_class_registrations"
             )
